src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -8,61 +8,126 @@ translate:
-  - /docs/accounts-partnerships/partnerships/partner-based-installation/amazon-web-services-aws-users
-  - /docs/accounts/install-new-relic/partner-based-installation/amazon-web-services-aws-users
-  - /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
-  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
-  - /docs/aws-cloudformation-integration
-  - /docs/infrastructure/amazon-integrations
-  - /docs/integrations/amazon-integrations
-freshnessValidatedDate: never
+    - /docs/integrations/amazon-integrations/get-started/introduction-aws-integrations
+    - /docs/infrastructure/amazon-integrations/getting-started/introduction-aws-integrations
+    - /docs/integrations/amazon-integrations/getting-started/introduction-aws-integrations
+    - /docs/accounts/install-new-relic/partner-based-installation/new-relic-aws-amazon-web-services
+    - /docs/partnerships/amazon-web-services-users
+    - /docs/accounts-partnerships/partnerships/partner-based-installation/amazon-web-services-aws-users
+    - /docs/accounts/install-new-relic/partner-based-installation/amazon-web-services-aws-users
+    - /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
+    - /docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
+    - /docs/aws-cloudformation-integration
+    - /docs/infrastructure/amazon-integrations
+    - /docs/integrations/amazon-integrations
+freshnessValidatedDate: 2024-09-20
 ---
 
-Amazon integrations let you monitor your AWS data in several New Relic features. Enabling the [AWS CloudWatch Metric Streams](/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream) integration is the recommended solution to monitor all CloudWatch metrics from all AWS services (including custom namespaces). On top of this, additional integrations are available to get extended visibility on key AWS services beyond the available CloudWatch metrics. For a full reference of the supported metrics, please check the available CloudWatch metrics for each service in the AWS documentation pages.
-
-## Connect AWS and New Relic [#connect]
-
-You can connect to AWS using [Amazon CloudWatch Metric Streams](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) or via [polling integration](/docs/infrastructure/amazon-integrations/connect/connect-aws-new-relic-infrastructure-monitoring/).
-
-Additional <DNT>**API Polling**</DNT> integrations can be enabled on top of the AWS CloudWatch metric streams in order to pull data that's not available as CloudWatch metrics. The following integrations are not replaced by the metric streams:
+New Relic offers several methods to collect performance data about your AWS environment. Once you've connected New Relic with AWS, you get total visibility into AWS telemetry data, letting you closely monitor the health of your AWS stack. 
```

Comment:
@akristen typically, we explode acronyms the first time we use them. That got me thinking about AWS. Is it simply so well known that we don't need to use Amazon Web Services (AWS) the first time? I'm on the fence about it, but wanted to raise it as something to think about.

Reply:
Otherwise, I think this is a solid rewrite of this first paragraph

Reply:
Tbh from a [sigh] SEO perspective it couldn't hurt. just seems like a win/win